### ISSUE TITLE
add charging limit service

### ISSUE THREAD

**Issue by lbbrhzn** [lbbrhzn/ocpp#51]
### Discussed in https://github.com/lbbrhzn/ocpp/discussions/48

<div type='discussions-op-text'>

<sup>Originally posted by **jeremycoppey** July  6, 2021</sup>
Would be nice to have some more info/sensors

- Charge time of session
- The energy usage for the current charging session
- Set charging limit
</div>

**Comment by drc38** [comment]
@lbbrhzn, will look at the custom service for the charging limit once the updated discovery / flow work is finished. The charger may need to registered as a platform/component as there is `async_register_entity_service` or `hass.services.register`, I'm not sure which one we need to use with the overall api structure?
